pymongo/mongo_client.py

```diff
@@ -1596,8 +1596,13 @@ def _process_kill_cursors(self):
             try:
                 self._cleanup_cursor(True, cursor_id, address, sock_mgr,
                                      None, False)
-            except Exception:
-                helpers._handle_exception()
+            except Exception as e:
```

Comment:
Please rename e to exc everywhere. We usually use exc as the name for exceptions.